### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by jimfromsa**
Docker build files and instructions for IOP location network containers

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+#  Path to log file. Optional, default value: ~/.iop-locnet/debug.log
+ENV LOGPATH  "/data/locnet/debug.log"
+
+#   Externally accessible IP address (ipv4 or v6) to be advertised for other nodes or clients. Required for seeds only, autodetected otherwise.
+#ENV HOST
+#   Internally accessible IP address that is required for mercury services on the same node.
+#ENV LOCALHOST
+
+# Host name of seed node to be used instead of default seeds. You can repeat this option to define multiple custom seed nodes.
```
Comment (izolyomi) [inline Dockerfile]: Bad comment, this is not the seed node but your own host id reported to other nodes of the network. Is this my mistake somehow or just some copy-paste problem you did? Also, there's a commented NODEID env var with the right description, one of them probably should be deleted.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+ENV NODEID  "YourNodeIdHere"
+
+# GPS latitude of this server as real number from range ((-90,90)
+ENV LATITUDE 48.2081743
+
+#  GPS longitude of this server as real number from range (-180,180)
+ENV LONGITUDE 16.3738189
```
Comment (izolyomi) [inline Dockerfile]: Do we really need default GPS values here which are not obviously uninitialized?

**Comment by izolyomi** [comment]
Thanks for the review request, @jimfromsa! I honestly don't have proper Docker skills to judge if there's everything Ok with the contents, so I'll trust you with this. As far as I understand, we had these docker details in a different repo so far and we're integrating them with the main repo here, right?

**Review by wigy-opensource-developer — APPROVED** [review]
@jimfromsa I am not that good at docker yet (although I am learning it day by day). What I saw in the last 2 commits made today, I liked. Seems like @izolyomi also has questions in parts that was already in a branch for a looooong time.

**Pull request merged**